.travis.yml

```diff
@@ -10,17 +10,18 @@ cache:
     - "/home/travis/.yarn-cache"
 
 before_install:
-  - "npm config set spin false"
-  - "npm --version"
+  - "yarn --version"
   - "phantomjs --version"
+  - "git clone https://github.com/ontohub/ontohub-backend ../ontohub-backend"
+  - "ln -s $(readlink -f ../ontohub-backend/spec/support/api/schemas) app/schemas"
```

Comment:
I'll create an issue after this PR is merged to only link the `models` and `relationships` directories during delopyment.